arelle/plugin/validate/DBA/rules/tr.py

```diff
@@ -17,12 +17,22 @@
 from . import lookup_namespaced_facts
 from ..PluginValidationDataExtension import PluginValidationDataExtension
 from ..ValidationPluginExtension import NAMESPACE_GSD
+from ..DisclosureSystems import (
+    ARL_2022_PREVIEW,
+    ARL_2024_PREVIEW,
+    ARL_2024_MULTI_TARGET_PREVIEW
+)
 
 _: TypeGetText
 
 
 @validation(
     hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        ARL_2022_PREVIEW,
+        ARL_2024_PREVIEW,
+        ARL_2024_MULTI_TARGET_PREVIEW
+    ],
```

Comment:
Are the only validations that need to specify their disclosure systems the ones that don’t run for multi-target? To simplify maintenance, it would be better to avoid explicitly specifying validations that apply to all disclosure systems, so we don’t have to update them annually. Additionally, we could define a list variable containing all single-target disclosure systems and reference it throughout for validations that don't run for multi-target, reducing the need to update individual rules next year if we expect them to stick around.